CRM/Utils/Check/Component/Env.php

```diff
@@ -942,35 +942,7 @@ public function checkMysqlVersion() {
     $minRecommendedVersion = CRM_Upgrade_Incremental_General::MIN_RECOMMENDED_MYSQL_VER;
     $mariaDbRecommendedVersion = '10.1';
     $upcomingCiviChangeVersion = '5.28';
-    if (version_compare(CRM_Utils_SQL::getDatabaseVersion(), $minInstallVersion, '<')) {
```

Comment:
Is this not useful as we start to kill 5.6 on whatever timeframe that is?

Reply:
@eileenmcnaughton last i checked I wasn't sure if the 5.6 time frame i.e. drop in 5.34 had been agreed to yet or not?

Reply:
Well I guess we've gone with implicit agreement for 5.5 drop so let's assume the timeline is generally agreed